lib/board.rb

```diff
@@ -25,10 +25,16 @@ def valid_coordinate?(coordinate)
     @cells.has_key?(coordinate)
   end
 
-  # def valid_placement?(ship, coordinates)
-  #   ship.length == coordinates.count && consecutive_check(ship, coordinates) == true
-    # require 'pry'; binding.pry
-  # end
+  def valid_placement?(ship, coordinates)
```

Comment:
When we go to refactor, lets try to figure out why we have to include (ship, coordinates) with every helper method.

Reply:
Agreed!